modules/update-preparing-list.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-list_{context}"]
+= Removed Kubernetes APIs
+
+{product-title} 4.12 uses Kubernetes 1.25, which removed the following deprecated `v1beta1` and `v2beta1` APIs. You must migrate manifests and API clients to use the `v1` or `v2` API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Kubernetes documentation].
+
+.`v1beta1` and `v2beta1` APIs removed from Kubernetes 1.25
+[cols="2,2,1",options="header",]
+|===
+|Resource |API |Notable changes
+
+|`CronJob`
+|`batch/v1beta1`
+|No
+
+|`EndpointSlice`
+|`discovery.k8s.io/v1beta1`
+|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#endpointslice-v125[Yes]
+
+|`Event`
+|`events.k8s.io/v1beta1`
+|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#event-v125[Yes]
+
+|`HorizontalPodAutoscaler`
+|`autoscaling/v2beta1`
+|No
+
+|`PodDisruptionBudget`
+|`policy/v1beta1`
+|link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125[Yes]
+
+|`PodSecurityPolicy`
+|`policy/v1beta1`
+|No
+
+|`RuntimeClass`
+|`node.k8s.io/v1beta1`
+|No
+
+|===
+
+// TODO: Do we need to say anything special about PodSecurityPolicy, since it's a little different and there isn't a v1 or v2? Anything OCP-specific on this compared to Kubernetes' instructions (because of PSA)?
```

Comment:
@sanchezl thoughts on this?

Reply:
I would love to say something about PodSecurity policy in the 4.12 release notes (e.g. see me breaking a fairly basic dummy workload [here|https://issues.redhat.com/browse/OTA-784]).  But I think we can decouple it from the Kube API deprecations, because the 4.11 exposure-checking pipelines are different.

Reply:
Sounds good, I'll leave this note here as a reminder, just to see if we do want to come back to say anything here for PSA.

Reply:
I think adding the links to the upstream docs in the "Resource | API Version | Migrate To | Notable Changes" tables is a good start. But I don't know enough to say if there are OpenShift specific docs that should be written or referenced[^1]. 

[^1]: I see that we have a [Understanding and managing pod security admission topic](https://docs.openshift.com/container-platform/4.11/authentication/understanding-and-managing-pod-security-admission.html), but I'm not familiar enough with it to comment.

Reply:
Thanks, I'll include links to both upstream and to our OCP section on it